app/parse.py

```diff
@@ -8,5 +13,26 @@
     duration: str
 
 
+def get_course(course: Tag) -> Course:
+    return Course(
+        name=course.select_one(".ProfessionCard_title__Zq5ZY > h3").text,
+        short_description=(
+            course.select_one(".typography_landingTextMain__Rc8BD.mb-32").text
+        ),
+        duration=(
+            course.select(".ProfessionCardTags_regularTag__yTc6K")[-1].text
+        )
+    )
+
+
 def get_all_courses() -> list[Course]:
-    pass
+    page = requests.get(BASE_URL).content
```

Comment:
Consider adding error handling for the HTTP request using `requests.get`. This could include handling exceptions such as `requests.exceptions.RequestException` to ensure the program doesn't crash if the request fails.